scripts/core/program.yml

```diff
@@ -169,18 +169,20 @@ params:
     - type: "$x_program_info_t"
       name: propName
       desc: "[in] name of the Program property to query"
-    - type: "size_t*"
+    - type: "size_t"
       name: propSize
       desc: |
-            [in,out] pointer to the size of the Program property.
-            If *propSize is 0 or greater than the number of bytes of the Program property,
-            the call shall update the value with actual number of bytes of the Program property.
+            [in] the size of the Program property.
     - type: "void*"
       name: propValue
       desc: |
-            [in,out][optional][range(0, *propSize)] value of the Program property.
-            If *propSize is less than the number of bytes for the Program property,
-            only the first *propSize bytes will be returned.
+            [in,out][optional][range(0, propSize)] value of the Program property.
+            If propSize is less than the number of bytes for the Program property,
```

Comment:
Here too, this behaviour differs to that of other info queries e.g. https://github.com/oneapi-src/unified-runtime/pull/64/files#diff-d87d905cff6868fb3e0d3aadb38e002f331e481878eaa7ffae36bcfb468ada93R89 which fails if there isn't enough memory, rather than returning `min(propSize, pPropSizeRet)` bytes.

Reply:
I think you're right 👍